aries-site/src/examples/templates/popover/PopoverInlineExample.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState, useRef, useContext } from 'react';
+import { Popover } from 'aries-core';
+import {
+  Anchor,
+  Grid,
+  Card,
+  CardBody,
+  Button,
+  Box,
+  Paragraph,
+  Text,
+  Heading,
+  ResponsiveContext,
+} from 'grommet';
+import { CircleInformation } from 'grommet-icons';
+
+const device = [
+  {
+    title: 'Require Assignments',
+    devices: '10',
```

Comment:
More realistic numbers.

```suggestion
    devices: '17',
```